soweego/linker/evaluate.py

```diff
@@ -9,55 +9,114 @@
 __license__ = 'GPL-3.0'
 __copyright__ = 'Copyleft 2019, Hjfocs'
 
+import json
 import logging
 import os
+from collections import defaultdict
 
```

Comment:
Import `sys` here to be used later on to define the application's exit code

```suggestion
import sys
```